src/js/components/Anchor/stories/Default.js

```diff
@@ -14,6 +13,4 @@ const Default = () => {
   );
 };
 
-storiesOf('Anchor', module).add('Default', () => <Default />, {
-  chromatic: { disable: true },
-});
+export const Default = () => <DefaultAnchor />;
```

Comment:
Chromatic should still be disabled on the story. To see an example of this look at Accordion/stories/Simple.js in this PR https://github.com/grommet/grommet/pull/4654

Reply:
Fixed :)